### DISCUSSION TITLE
Appreciation thread for Ansible integration

### DISCUSSION THREAD

**Discussion by MartyLake** [xavierog/moulti#22] [General]
Hello,
I started learning Ansible recently, and I just tried moulti. The result is so beautiful !!!
Thanks for making such a package.

**Comment by xavierog** [comment]
Hi,

> The result is so beautiful !!!

Is this result worth a screenshot, by chance?
I have checked both Github issues and discussions and noticed that, with the exception of a single 3-line screenshot, I remain the author of **all** Moulti screenshots.

Tip: Flameshot is a nice tool if you want to obfuscate hostnames or other parts of the screenshot.
